sdk/communication/Azure.Communication.CallAutomation/src/Models/DialogOptionsInternal.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Communication.CallAutomation
+{
+    [CodeGenModel("DialogOptions")]
+    public partial class DialogOptionsInternal
```

Comment:
you didnt actually make this internal

Reply:
Updated

Reply:
Had to expose DialogStateResponse before the error was fixed. All internal objects are actually internal now